jakelib/pipeline.jake

```diff
@@ -33,36 +34,40 @@ namespace('pipeline', function () {
 	desc('Deploy via pipeline. | [\'env\',\'cluster\',\'repo\',\'branch\']');
 	task('deploy', ['aws:loadCredentials'], { async: true }, function(environment, cluster, repo, toBranch) {
 
-    try {
-      let a = CLUSTERS[environment][cluster].protected;
-    } catch (e) {
-      throw new Error(`environment and cluster MISMATCH "${cluster}" is not a valid "${environment}" cluster`)
+    if (environment !== 'eph') {
+      try {
+        let a = CLUSTERS[environment][cluster].protected;
+      } catch (e) {
+        throw new Error(`environment and cluster MISMATCH "${cluster}" is not a valid "${environment}" cluster`)
+      }
+  
+      const IS_PROTECTED_TARGET = CLUSTERS[environment][cluster].protected;
+      const IS_RELEASE_BRANCH = toBranch.includes('release/') || toBranch.includes('hotfix/');
```

Comment:
didn't realize that this was the actual check for release branches.  seems important to know that a typo in `hotfix/` or `release/` will block a deployment!  😅